services/entity.lua

```diff
@@ -8,6 +8,30 @@ local RegisterSprites = require 'systems/RegisterSprites'
 
 local entity_directory = 'entities'
 
+--------Jon's shiplist stuff--------------- 
+local shipList = require('ships/shipList') -- List of ships and attributes J.R.C 2/1/22
+
+local function get_ship(index)
+    print("player.lua ship_index: " .. index)
+    return shipList[index]
+end
+
+-- "Caveman way" of setting the player ship attributes from map - J.R.C 2/1/22
+local function check_ship_type(object, entity)
+    -- Gross code to Check the ship_type of each applicable entity and set the player attributes accordingly - J.R.C 2/1/22
+    if object.ship_type ~= nil then
```

Comment:
Another short Lua syntax tip is that instead of using this, you can use:
`if object.ship_type then`
This in lua essentially means:
`if foo ~= nil and foo ~= false then`